docker/mongodb/scripts/run.sh

```diff
@@ -1,22 +1,31 @@
 #!/bin/bash
 
+
+run_mongo_commandline () {
+if (( $(echo "$MONGO_MAJOR > 5.9" | bc -l) )); then
```

Comment:
Instead of depending on `bc`, how about just doing a numeric comparison of the first digit?  Something like:
```
major=$(echo "$MAJOR_VERSION" | cut -d '.' -f1)
if [ $major -gt 5 ]; then
...
```

Reply:
easier solution. Also bc for some reason is not getting installed in the 4.4 version.